src/site/components/navigation-sidebar-trigger.html

```diff
@@ -7,40 +7,12 @@
 Used to open or close the side bar navigation bar on mobile screens. Must be
 used in a template with navigation-sidebar.html
 
-{% endcomment %}
-{% if buildtype == 'vagovprod' %}
   <button type="button" class="va-btn-sidebarnav-trigger vads-u-width--full"
     aria-controls="va-detailpage-sidebar">
     <span>
       <b>More in this section</b>
       {% include "src/site/assets/img/arrow-right-white.svg" %}
     </span>
   </button>
+{% endcomment %}
```

Comment:
Rather than just comment out the markup in this file, can the file be deleted entirely?  I count 25 files where it is included.

Also, I think we can remove this:

https://github.com/department-of-veterans-affairs/vets-website/blob/7db9b1d9b33b92d9c2fa0f46e605be039e6e0c30/src/platform/site-wide/sass/modules/_m-layers.scss#L37-L39